cli/src/commands/content/addCuratorToGroup.ts

```diff
@@ -17,13 +20,32 @@ export default class AddCuratorToGroupCommand extends ContentDirectoryCommandBas
   ]
 
   static flags = {
+    permissions: flags.build({
+      parse: (value: string) => {
+        const arr = value.split(/[, ]+/).map((v) => {
+          const possibleActions = keysOf<ChannelActionPermission, 'PalletContentChannelActionPermission'>(
+            'PalletContentChannelActionPermission'
+          )
+          if (!possibleActions.includes(v as ChannelActionPermission['type'])) {
+            throw new Error(`Expected comma-separated permissions[${possibleActions}], but received: ${value}`)
+          }
+          return v as ChannelActionPermission['type']
+        })
+        return arr
+      },
+    })({
+      char: 'p',
+      description: `Comma separated list of permissions to associate with the curator`,
+      required: true,
+    }),
```

Comment:
```suggestion
    permissions: flags.enum({
      options: keysOf<ChannelActionPermission, 'PalletContentChannelActionPermission'>(
        'PalletContentChannelActionPermission'
      ),
      multiple: true,
      char: 'p',
      description: `Comma separated list of permissions to associate with the curator`,
      required: false,
    }),
```

And later on you can use:
```typescript
      createType(
        'BTreeSet<PalletContentChannelActionPermission>',
        typeof permissions === 'string' ? [permissions] : permissions
      ),
```

Reply:
addressed in [6adbb26](https://github.com/Joystream/joystream/pull/4084/commits/6adbb26c9931578c08b9e5edb8d163f55175ada0)